tests/integration/test_dummy_integration.py

```diff
@@ -0,0 +1,59 @@
+from __future__ import annotations
+
+from pathlib import Path
+
+import pytest
+from helpers import PROTEUS_ROOT, resize_to_match
+from matplotlib.testing.compare import compare_images
+from pandas.testing import assert_frame_equal
+
+from proteus import Proteus
+from proteus.utils.coupler import ReadHelpfileFromCSV
+
+IMAGE_LIST = (
+        "plot_elements.png",
+        "plot_escape.png",
+        "plot_fluxes_global.png",
+        "plot_global_lin.png",
+        "plot_global_log.png",
+        "plot_observables.png",
+        "plot_sflux.png",
+        )
+
+@pytest.fixture
+def test_dummy_run():
+    config_path = PROTEUS_ROOT / 'input' / 'dummy.toml'
+    ref_output = PROTEUS_ROOT / 'tests' / 'data' / 'integration' / 'dummy'
+
+    runner = Proteus(config_path=config_path)
+
+    runner.start()
+
+    output = Path(runner.directories['output'])
+
+    hf_all = ReadHelpfileFromCSV(output)
+    hf_all_ref = ReadHelpfileFromCSV(ref_output)
+
+    assert_frame_equal(hf_all, hf_all_ref, rtol=1e-2)
+
+def test_plot_dummy_integration(test_dummy_run):
+
+    out_dir = PROTEUS_ROOT / 'output' / 'dummy'
+    ref_dir = PROTEUS_ROOT / 'tests' / 'data' / 'integration' / 'dummy'
+
+    for image in IMAGE_LIST:
```

Comment:
Consider splitting these tests using `pytest.mark.parametrize`. See examples here: https://docs.pytest.org/en/7.1.x/how-to/parametrize.html#pytest-mark-parametrize-parametrizing-test-functions

Reply:
This is what I used at first instance, but the problem is that with parametrize it re-executes the fixture (the integration test) for each image, which does not make sense.

Reply:
You can set the fixture to only run once using `scope='module'`

Reply:
Done, thanks for the tip.